### PR TITLE
Add `rts/Benchmarking.md`

### PR DESCRIPTION
**Pull request by osa1**

### CODE REVIEW

**Comment by osa1** [comment]
@ggreif @crusso please feel free to edit the file directly in my branch or suggest changes.

File: rts/Benchmarking.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  before running drun.
+- Copy generated .csv file after each run, give it a descriptive name.
+
+After that you will have csv files that you can compare.
+
+## How to compare CSV files
+
+Most of the time I simply do `nvim -d <csv1> <csv2>`.
+
+I have a Rust program to generate plots for instructions, dirtied pages etc.
```
Comment (ggreif) [inline rts/Benchmarking.md:144]:
```suggestion
I have a Rust program to generates plots for instructions, dirtied pages etc.
```

Comment (ggreif) [reply rts/Benchmarking.md:144]: Oh, this sounds interesting...

**Review by ggreif — APPROVED** [review]

File: rts/Benchmarking.md (hunk 1 above)
Comment (osa1) [reply rts/Benchmarking.md]: https://github.com/osa1/generate_drun_bench_plots

**Comment by ggreif** [comment]
Thanks! I have added it:
https://github.com/dfinity/motoko/commit/1b1cc79b6f4628c20c876164e1c58a20c54367bf

**Pull request merged**